src/components/welcome/MyAccounts/SubAccountItem.tsx

```diff
@@ -55,11 +55,16 @@ const SubAccountItem = ({ onLinkClick, safeItem, safeOverview }: SubAccountItem)
 
   const cfSafeSetup = extractCounterfactualSafeSetup(undeployedSafe, chain?.chainId)
 
+  const onSwitchSafe = () => {
+    trackEvent({ ...OVERVIEW_EVENTS.SWITCH_NETWORK, label: chainId })
```

Comment:
I switch network the correct event here?
I could also switch to another Multichain Safe and therefore switch Safe and network or only switch to another Safe on the same network.

Maybe we should call it "choose Safe from sidebar" or something like that instead?

Reply:
It should be fine here since its in the `SubAccountItem` so only emitted if the user switches between the same safe on a different network. I added this to match the network selector in the header which also allows switching between the same safe on different networks.

Reply:
But I can open a `SubAccountItem` of a different Safe. 
So for instance switch from Safe "0x00...01" on Gnosis Chain to Safe "0x00...02" on Polygon. 

Also now its impossible to distinguish between switching networks in the sidebar or through the network selector.

Reply:
Good point! Maybe we should leave it out then since we already emit an `Open Safe` when a Safe is opened from the Sidebar.